src/DataStruct.hs

```diff
@@ -21,11 +22,11 @@ data Document = Document {
     }
 
 data Header = Header {
-    title :: String,
-    author :: Maybe String,
-    date :: Maybe String
+    contents :: [HeaderElement]
```

Comment:
Why this change, it's quite more permissive, now we can have 5 titles or 3 authors and no title. Can you explain this decision?

Reply:
we shouldnt have multiple same attributs as long as we don't authorize it in the parsing, as the whys and wherefores, i redirect you to @Marius-P1, i simply follow orders (its to keep an order of what attribut come before another)